Plugins/NativeEngine/Source/NativeEngine.cpp

```diff
@@ -321,7 +321,7 @@ namespace Babylon
             JS_CLASS_NAME,
             {
                 // This must match the version in nativeEngine.ts
-                StaticValue("PROTOCOL_VERSION", Napi::Number::From(env, 8)),
+                StaticValue("PROTOCOL_VERSION", Napi::Number::From(env, 9)),
```

Comment:
Is this necessary? Can we modify the code to do the old behavior if instanceCount is missing?

Reply:
Changed the PR to avoid requiring protocol version changes.